apps/fabric-website/src/pages/Styles/FabricIconsPage/FabricIconsPage.tsx

```diff
@@ -1,10 +1,21 @@
 import * as React from 'react';
+import { Pivot, PivotItem } from 'office-ui-fabric-react';
 import { IconGrid } from '../../../components/IconGrid/IconGrid';
 import { IPageSectionProps } from '@uifabric/example-app-base/lib/index2';
 import { IStylesPageProps, StylesAreaPage } from '../StylesAreaPage';
 import { FabricIconsPageProps } from './FabricIconsPage.doc';
 import * as styles from './FabricIconsPage.module.scss';
 import { Platforms } from '../../../interfaces/Platforms';
+import * as IconNames from '@uifabric/icons/src/IconNames';
+
+const allFabricIcons: { name: string }[] = [];
+// IconNames is a const enum, so use the following as a workaround to iterate over it.
+// tslint:ignore-next-line no-any
+for (const iconName in (IconNames as any).IconNames) {
```

Comment:
I don't think this will even work at runtime? Const enums are compiled out.

Reply:
Correct. This method worked in vr-tests, but it's not working here at build time (does work when running dev-server, though).